collectors/channels_collector.go

```diff
@@ -51,6 +53,11 @@ type ChannelsCollector struct {
 	// errChan is a channel that we send any errors that we encounter into.
 	// This channel should be buffered so that it does not block sends.
 	errChan chan<- error
+
+	// cache is a map storing results from a ticker to reduce grpc server
+	// load on lnd
+	cache      map[string]interface{}
```

Comment:
I think just one thing cached rn so we can shed `interface{}` in favor of the actual type.

Reply:
Yep, refactored this.